ee/common-annotations-1.3.adoc

```diff
@@ -0,0 +1,49 @@
+= Common Annotations 1.3  
+:author:            Tomaz Cerar
+:email:             tcerar@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          comma,separated,tags
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+
+Commons Annotations 1.3 is a small change update to 1.2 MR.
+Main change is allowing @Repeatble on @DatasourceDefintion and @Priority annotation is allowed to also on methods now.
+We are using our fork of upstream's https://github.com/javaee/javax.annotation
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/EAP7-900[EAP7-900]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/WFLY-6463[WFLY-6463]
+* https://issues.jboss.org/browse/WFLY-9748[WFLY-9748]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+=== Affected Projects or Components
+
+* WildFly
+* Weld Integration
+
+== Requirements
+
+Add conditional support for Common Annotations 1.3 This will be controlled by the `ee8.preview.mode` system property. If
+the property is not present or not set to `true` Comon Annotations 1.2 will be used instead.
+
+We need to check if Weld works fine with @Priority changes, which is tracked in https://issues.jboss.org/browse/WFLY-9748
```

Comment:
The issue is wrong. The correct issue is https://issues.jboss.org/browse/WELD-2239